docs/DeveloperGuide.md

```diff
@@ -1,34 +1,255 @@
+`DaySchedulePair` class is implemented to act as a pair between a day input String and schedule ArrayList.
+Additionally, they implement the following operations:
+
+`addDailyRoute(String ,ArrayList<String>)` — Maps the inputted day string to the inputted ArrayList of the schedule of the day in a hashmap . <br />
+`getDailyRoute(String)` — Returns the schedule of the day that is mapped to the inputted day. <br />
+
+These operations are exposed in the `DailyRoute` class  as `DailyRoute#addDailyRoute()` and `DailyRoute#getDailyRoute(String)`. <br />
+Given below is an example usage scenario and how the addDailyRoute mechanism behaves at each step. <br />
+Step 1. The user launches the application.<br />
+Step 2. The user executes `add daily route` command. UI will then prompt the user `SELECT ENTRY TO ADD:`  to input a day index. <br />
+Step 3. The UI then prompts the user to input the next block that is in the day's schedule.  <br /> The inputted location will be appended to an ArrayList. <br />
+Step 4. Repeat step 3 until the word `END` is input by the user. <br /> 
+Step 5. The inputted day, and the filled Arraylist from step 3 is then passed into the DailyRoute object<br /> This done using the addDailyRoute method from the DailyRoute class <br />
+Step 6. The day and filled Arraylist passed in step 5 is then saved in a hashmap that the DailyRoute object contains. <br /> 
+
+The following image shows the sequence diagram in which the add day command is implemented
+![img.png](addday.png)
+
+Given below is an example usage scenario and how the showDailyRoute mechanism behaves at each step.
+
+Step 1. The user launches the application.<br />
+Step 2. The user executes `show daily route` command. UI will then prompt the user `SELECT ENTRY TO VIEW:`  to input a day index. This returns an arraylist of the day's schedule <br />
+Step 3. The routing algorithm is now performed for each of the blocks in the array list in order. Each execution of the routing algorithm returns a string which is then appended to the previous one. <br />
+Step 4. The String is output through Daily Route Ui <br />
+
+The following image shows the sequence diagram in which the add day command is implemented
+![img.png](showday.png)
+
+### 3.3. Finding The Shortest Route Feature
+#### Current Implementation
+
+The current implementation of finding the shortest route is facilitated by the `Router` class which uses data stored in `NusMap`, `Block`, and `BlockAlias` class to return the shortest path.
+
+The `GoCommand` class extends the `Command` class and overrides the `execute` method to run the routing algorithm.
+
+The image below depicts how the `GoCommand` is implemented.
+
+![img.png](router.png)
```

Comment:
maybe this is a non standard UML notation for the diagram?

Reply:
![image](https://user-images.githubusercontent.com/59407923/113076871-8b531580-9202-11eb-824b-786f8691017f.png)